packages/electron-updater/src/util.ts

```diff
@@ -35,3 +36,16 @@ export function blockmapFiles(baseUrl: URL, oldVersion: string, newVersion: stri
   const oldBlockMapUrl = newUrlFromBase(`${baseUrl.pathname.replace(new RegExp(escapeRegExp(newVersion), "g"), oldVersion)}.blockmap`, baseUrl)
   return [oldBlockMapUrl, newBlockMapUrl]
 }
+
+export function execShellCommand(cmd: string): Promise<{ stdout: string | Buffer; stderr: string | Buffer }> {
```

Comment:
I was looking further into this util function and I think it may be duplicating 
https://github.com/electron-userland/electron-builder/blob/d307c21b34abee1cd2b8dcd3e705995914dda70f/packages/builder-util/src/util.ts#L208-L212

Is it possible to use `util.spawn(...)` for this operation instead of creating a new `execShellCommand`

Reply:
Do I have access to electron-builder utils from electron-updater package?

Reply:
Ahh,

```
import { spawn } from "builder-util"
```

I see it now :)